third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -1017,6 +1017,14 @@ IFACEMETHODIMP AXPlatformNodeWin::get_accName(VARIANT var_id, BSTR* name_bstr) {
 
   bool has_name = target->HasStringAttribute(ax::mojom::StringAttribute::kName);
   std::u16string name = target->GetNameAsString16();
+
+  // Simply appends the tooltip, if any, to the end of the MSAA name.
+  const std::u16string tooltip =
+      target->GetString16Attribute(ax::mojom::StringAttribute::kTooltip);
+  if (!tooltip.empty()) {
+    AppendTextToString(tooltip, &name);
+  }
```

Comment:
It looks like UIA has better support for tooltips via the `UIA_HelpTextPropertyId` property. At some point, we should plan to do some research into how much effort would be required to migrate from MSAA to UIA, but this seems like a reasonable approach given the constraints of MSAA.